build.sbt

```diff
@@ -2,7 +2,7 @@ import Dependencies._
 
 lazy val `akkaserverless-java-sdk` = project
   .in(file("."))
-  .aggregate(sdk, testkit, tck, codegenCore, codegenJava, codegenJavaCompilationTest, samples)
+  .aggregate(sdk, testkit, tck, codegenCore, codegenJava, codegenJavaCompilationTest)
 
```

Comment:
Can more things be removed/simplifed in the build? I think stuff related to this was added in https://github.com/lightbend/akkaserverless-java-sdk/pull/188
e.g. ReflectiveCodeGen.scala

Hopefully the problem with that one has to `clean` after changing `codegenJava` will go away? That is more of a time sink than `publishM2`.

Reply:
`codegenJavaCompilationTest` (#188) is separate from `valueentity-generated`. The problem that one has to `clean` after changing `codegenJava` is related to `codegenJavaCompilationTest`, not to `valueentity-generated`.

Reply:
ok, thanks for clarifying